web-api-db-ctrl/elastic-search/mapping.json

```diff
@@ -0,0 +1 @@
+../../web-api/elastic-search/mapping.json
```

Comment:
did github always display symlinks this way or is something broken?

After the update of the integration tests we will only have the mapping in `web-api-db-ctrl` and the one in `web-api` will go away as it and the create_es_index docker container are no longer needed.